src/Marketplace/Controller.php

```diff
@@ -86,7 +86,7 @@ public static function getAPI(): PluginsApi
      *
      * @return bool
      */
-    public function downloadPlugin(): bool
+    public function downloadPlugin($skip_install = false): bool
```

Comment:
Double negatives are often subject to issues.
I propose to use `$auto_install = true`.

Also, you should force the parameter type to boolean and add corresponding doc.